mimic/rest/fastly_api.py

```diff
@@ -11,6 +11,18 @@
 from mimic.canned_responses import fastly
 
 
+def text_urldata(urldata):
+    """
+    Convert ``urldata`` into text.
+
+    :param urldata: a dictionary mapping bytes to lists of bytes.
+
+    :return: a dictionary mapping text to lists of text.
+    """
+    return dict([(k.decode("utf-8"), [vv.decode("utf-8") for vv in v])
+                 for [k, v] in urldata.items()])
```

Comment:
Oh also, re: the "unpacked into a list" thing; unpacking is just into variables, so:

```python
x, y = 1, 2
[x, y] = 1, 2
(x, y) = 1, 2
```

are all semantically equivalent.  The reason I often choose square brackets rather than no brackets or parentheses is that they behave more consistently when you shrink the unpack to a size of 1, like so:

```python
unpackable = [1, 2]
[x, y] = unpackable
(x, y) = unpackable
# both x=1, y=2. but:
unpackable = [1]
[x] = unpackable # x = 1
(x) = unpackable # x = [1]
```
So especially when I'm bulk editing lots of code (as in this branch) I try to stick with square brackets in case I accidentally robotically remove the last element from some tuple I'm unpacking by accident; I either want to get an error (if the length is now wrong) or a consistent unpacking result (if I'm unpacking other 2-tuples and 3-tuples nearby).

Hopefully this makes sense? :)